.idea/gradle.xml

```diff
@@ -15,7 +15,6 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
-        <option name="useQualifiedModuleNames" value="true" />
       </GradleProjectSettings>
     </option>
   </component>
```

Comment:
I think this file should be ignore from git. Perhaps we need clear git cache to apply changes

Reply:
Yes you are right, I made the issue #52 to solve that problem.